lib/std/fs.zig

```diff
@@ -686,21 +686,28 @@ pub const Dir = struct {
             return self.openFileW(path_w.span(), flags);
         }
 
+        var os_flags: u32 = os.O_CLOEXEC;
```

Comment:
I prefer the way of writing the code without `var` but it's not a merge blocker

Reply:
I tried to at first; but it was getting unwiedly and zig fmt kept making it look awful.